### PR TITLE
add copy assigning mechanism to modals

### PR DESCRIPTION
**Pull request by phoebemirman**
part of https://github.com/AugurProject/augur/issues/4715

### CODE REVIEW

File: packages/augur-ui/src/modules/create-market/components/common.tsx [hunk 1]
```diff
@@ -55,6 +55,7 @@ import {
   DISMISSABLE_NOTICE_BUTTON_TYPES,
 } from 'modules/reporting/common';
 import PreviewMarketTitle from 'modules/market/components/common/PreviewMarketTitle';
+import { StringNullableChain } from 'lodash';
```
Comment (bthaile) [inline packages/augur-ui/src/modules/create-market/components/common.tsx]: We decided a while ago not to use lodash in the UI. They have had mem issues in the past.

**Review by bthaile — CHANGES_REQUESTED** [review]
Remove lodash usage

File: packages/augur-ui/src/modules/create-market/components/common.tsx (hunk 1 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/create-market/components/common.tsx]: don't know how that got there. it isnt used

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🧡
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/jvghomz**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Review by bthaile — APPROVED** [review]
Works well. Lgtm

**Pull request merged**